From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: NotStevy <notstevy@ultrabuildmc.de>
Date: Tue, 20 Aug 2024 18:25:50 +0200
Subject: [PATCH] Build changes

Fix dev bundle not including :core and :api project, add cli build tools for uploading to api

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 0000000000000000000000000000000000000000..97e78e27ee0eea2c8b24886eeb19164d552323fe
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1,40 @@
+.gradle/
+
+# Eclipse stuff
+/.classpath
+/.project
+/.settings
+/.checkstyle
+/.factorypath
+
+# netbeans
+/nbproject
+/nb-configuration.xml
+
+# we use maven!
+/build.xml
+
+# maven
+/target
+
+# vim
+.*.sw[a-p]
+
+# various other potential build files
+/build
+/bin
+/dist
+/manifest.mf
+
+# Mac filesystem dust
+.DS_Store
+
+# intellij
+*.iml
+*.ipr
+*.iws
+.idea/
+
+# vs code
+/.vscode
+/.factorypath
diff --git a/build.gradle.kts b/build.gradle.kts
index 3a1cad6046387bacf0e7ef83be8ee2e1a92137b7..e952b11dc36899d3d5a09e6c8d488b81e4eff149 100644
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -1,8 +1,9 @@
 plugins {
+    `java-library`
 }
 
 dependencies {
     compileOnly(project(":api"))
-    compileOnly(project(":slimeworldmanager-api"))
+    compileOnly(project(":kitpvpslime-api")) // kitpvp rename
     implementation("com.github.luben:zstd-jni:1.5.2-2")
 }
